src/device_trezor/device_trezor.cpp

```diff
@@ -697,13 +693,13 @@ namespace trezor {
     unsigned device_trezor::client_version()
     {
       auto trezor_version = get_version();
-      if (trezor_version <= pack_version(2, 0, 10)){
-        throw exc::TrezorException("Trezor firmware 2.0.10 and lower are not supported. Please update.");
+      if (trezor_version < pack_version(2, 4, 3)){
+        throw exc::TrezorException("Minimal Trezor firmware version is 2.4.3. Please update.");
```

Comment:
From [these comments](https://github.com/trezor/trezor-firmware/pull/2232#issuecomment-1112419292):

> Another thing, I hard-coded to the monero wallet that firmware version 2.5.1 supports HF15

> We are releasing 2.5.1 which does not include this PR. Please bump to 2.5.2 in the monero PR.

I'm assuming only Trezor v2.5.2 will support bp+ and view tags. Wouldn't that mean the minimal Trezor version should be v2.5.2? Is the minimum set at v2.4.3 to support tx construction pre-fork?

Reply:
Nevermind, I see. Once this ships, Trezors v2.4.3 and up will be able to use those wallets until the fork

Reply:
Thanks for the review @j-berman! That was the idea, so old Trezors could still use the wallet even if they did not update yet. We can later enforce minimal version 2.5.2.